src/ZipkinBundle/Components/Messenger/ZipkinHandlerInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Envelope;
+use Zipkin\Tracing;
+
+interface ZipkinHandlerInterface
```

Comment:
I am not 100% we need this interface as the implementors are both handler and receiver which seem to not to have things in common more than the method name. I'd remove it.

Reply:
Ok